CMakeLists.txt

```diff
@@ -33,6 +36,7 @@ else()
     include(${QT_USE_FILE})
     include_directories(${QT_INCLUDES})
     set(QTCORE_LIBRARIES ${QT_QTCORE_LIBRARY})
+    set(QUAZIP_LIB_VERSION_PREFIX Qt)
```

Comment:
This is already breaking with existing convention and needs rebuild of all consumers, so you might as well choose Qt4 here to avoid confusion. Not that a lot of people will notice, as Qt4 is pretty much dead.